### ISSUE TITLE
Enhancement: Export only Visible layers

### ISSUE THREAD

**Issue by cilex-ft** [jespino/inkscape-export-layers#24]
The plugin shoud have a checkbox to "Export only visible layers", so it would be much easier to switch backgrounds.

**Comment by dmitry-t** [comment]
But you can leave backgrounds non-tagged (neither `[export]` nor `[fixed]`, isn't it sufficient?

**Comment by cilex-ft** [comment]
Not really... I have an image with 8 layers, making 2 "sets" of 4 layers each to output 2 final images. So to switch between "virtual image 1" and "virtual image 2" I check / uncheck layers, then I have to change all tags to export them. 

One should be able to define `[export]` and `[fixed]` tags, but only visible ones should be exported.
